.github/workflows/linux.yml

```
@@ -0,0 +1,84 @@
+---
+name: Linux CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  tests:
+    name: "${{ matrix.env.FLAVOR }} ${{ matrix.env.BOOST }} ${{ matrix.env.COMPILER }}"
+    runs-on: "ubuntu-latest"
+
+    container:
+      image: bondciimages.azurecr.io/ubuntu-1604:build-32869346
```

Did you try specifying the volumes here to get the source code under /root/bond like the build scripts are expecting? This may mean that the `sed` on the paths won't be needed.

```suggestion
      image: bondciimages.azurecr.io/ubuntu-1604:build-32869346
      volumes:
        - /github/workspace:/root/bond
```

(It looks like the source code gets cloned into `/github/workspace`.)

---

it didn't work, not sure why. I also tried - /home/runner/work/bond:/root since in the initialize container step it references /home/runner/work/ as the volume that later becomes the workspace

---

OK. Thanks for experimenting.